### PR TITLE
LPS-119092

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @jbalsas,

Attached is an update for http://issues.liferay.com/browse/LPS-119092.

Please let me know if you have any questions. Thanks!

### CODE REVIEW

File: modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/personal_menu/js/PersonalMenu.es.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+info: using fallback config in /Volumes/Projects/liferay/master/portal/modules/.prettierrc.js
```
Comment (jbalsas) [inline modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/personal_menu/js/PersonalMenu.es.js:1]: This looks odd...

File: modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/personal_menu/js/PersonalMenu.es.js [hunk 2]
```diff
@@ -34,8 +35,8 @@ function PersonalMenu({
 	function preloadItems() {
 		if (!preloadPromise.current) {
 			preloadPromise.current = fetch(itemsURL)
-				.then((response) => response.json())
-				.then((items) => setItems(items));
+				.then(response => response.json())
```
Comment (jbalsas) [inline modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/personal_menu/js/PersonalMenu.es.js:38]: Formatting looks odd as well, try running `yarn format` or `gradlew formatSource` from within the module to get it fixed. This looks like your IDE prettier settings taking over ours.

**Comment by jbalsas** [comment]
Hey @jonmak08, @wanderlast, thanks for sending this over!

I added a few quick comments on the PR, but seeing as this is `product-navigation-taglib`, I think the right team should be
@liferay-echo. Could you make sure this PR finds their way?

For future reference as well we're following a different review workflow from Frontend Infra. Please, take a look at the following for future PRs to the team.
- [Frontend Infrastructure Team code review process](https://github.com/liferay/liferay-frontend-guidelines/blob/master/dxp/code_review.md)
- [Pull Request Etiquette](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/pull_requests.md)

Thanks! :)

**Comment by jonmak08** [comment]
New Pull Request: https://github.com/liferay-echo/liferay-portal/pull/1914

**Pull request closed without merging**